ParselyTrackerTests/ParselyTrackerTests.swift

```diff
@@ -22,72 +22,57 @@ class ParselyTrackerTests: ParselyTestCase {
-        XCTAssert(testUrlAccumulator.isEngaged,
-                  "After a call to Parsely.startEngagement, the internal accumulator for the engaged url should exist " +
-                  "and its isEngaged flag should be set")
+        // After a call to Parsely.startEngagement, the internal accumulator for the engaged url should exist
+        // and its isEngaged flag should be set
+        expectParselyState(self.parselyTestTracker.track.engagedTime.accumulators[self.testUrl]?.isEngaged).toEventually(beTrue())
     }
     func testStopEngagement() {
         parselyTestTracker.startEngagement(url: testUrl)
-
         expectParselyState(self.parselyTestTracker.track.engagedTime.accumulators[self.testUrl]).toEventuallyNot(beNil())
 
         parselyTestTracker.stopEngagement()
+        // After a call to Parsely.startEngagement followed by a call to Parsely.stopEngagement, the internal
+        // accumulator for the engaged url should exist and its isEngaged flag should be unset
         expectParselyState(self.parselyTestTracker.track.engagedTime.accumulators[self.testUrl]?.isEngaged).toEventually(beFalse())
-
-        let internalAccumulators:Dictionary<String, Accumulator> = parselyTestTracker.track.engagedTime.accumulators
-        let testUrlAccumulator: Accumulator = internalAccumulators[testUrl]!
-        XCTAssertFalse(testUrlAccumulator.isEngaged,
-                  "After a call to Parsely.startEngagement followed by a call to Parsely.stopEngagement, the internal " +
-                  "accumulator for the engaged url should exist and its isEngaged flag should be unset")
     }
-    func testTrackPlay() {
+    func testTrackPlay() throws {
         parselyTestTracker.trackPlay(url: testUrl, videoID: testVideoId, duration: TimeInterval(10))
-        expectParselyState(self.parselyTestTracker.track.videoManager.trackedVideos.isEmpty).toEventually(beFalse())
+        // After a call to parsely.trackPlay, there should be exactly one video being tracked
+        expectParselyState(self.parselyTestTracker.track.videoManager.trackedVideos.count).toEventually(equal(1))
 
-        let videoManager: VideoManager = parselyTestTracker.track.videoManager
-        let trackedVideos: Dictionary<String, TrackedVideo> = videoManager.trackedVideos
-        XCTAssertEqual(parselyTestTracker.eventQueue.length(), 1,
-                       "A call to Parsely.trackPlay should add an event to eventQueue")
-        XCTAssertEqual(trackedVideos.count, 1,
-                       "After a call to parsely.trackPlay, there should be exactly one video being tracked")
-        let testVideo: TrackedVideo = trackedVideos.values.first!
-        XCTAssert(testVideo.isPlaying,
-                  "After a call to Parsely.trackPlay, the tracked video should have its isPlaying flag set")
+        XCTAssertEqual(parselyTestTracker.eventQueue.length(), 1, "A call to Parsely.trackPlay should add an event to eventQueue")
+        try XCTAssert(
```

Comment:
Nitpick: I find `XCTAssertTrue` clearer than `XCTAssert`, even though they do the same thing.

<img width="482" alt="Screenshot 2023-04-17 at 2 06 48 pm" src="https://user-images.githubusercontent.com/1218433/232375612-bd2aa4cb-0b34-496a-99c1-e9bbebaab649.png">

```suggestion
        try XCTAssertTrue(
```